<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyCalculator</string>
  </property>
  <property name="windowIcon">
   <iconset theme="accessories-calculator"/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="btn_ac">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(128, 128, 128, 0.8);
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color :rgb(77, 77, 77);
border-top-left-radius: 10px; 
border-top-right-radius: 10px;
font: 700 16pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;

</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_plus_minues">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(128, 128, 128, 0.8); 
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num_9">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #808080; 
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #808080; 
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #808080; 
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #808080; 
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #808080; 
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #808080; 
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #808080; 
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #808080; 
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #808080; 
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dot">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #808080; 
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_num_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #808080; 
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
border-bottom-left-radius: 10px; 
margin-bottom: 0px;
padding-bottom: 0px;

</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sum">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgb(76, 159, 46);
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_div">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(76, 159, 46);
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_result">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(76, 159, 46);
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
border-bottom-right-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mul">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(76, 159, 46); 
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sub">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgb(76, 159, 46);
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(76, 159, 46);
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cot">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(76, 159, 46);
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>cot</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tan">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgb(76, 159, 46);
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cos">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(76, 159, 46); 
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_log">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(76, 159, 46);
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sqrt">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(128, 128, 128, 0.8);
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>√x</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_percent">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(128, 128, 128, 0.8);
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ce">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(128, 128, 128, 0.8); 
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sqr">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(128, 128, 128, 0.8);
font: 700 14pt &quot;Consolas&quot;;
font-weight: bold; 
color: white; 
border: 1px solid white;
</string>
    </property>
    <property name="text">
     <string>x²</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>btn_ac</zorder>
   <zorder>txtbox</zorder>
   <zorder>btn_num_9</zorder>
   <zorder>btn_num_7</zorder>
   <zorder>btn_num_8</zorder>
   <zorder>btn_num_6</zorder>
   <zorder>btn_num_4</zorder>
   <zorder>btn_num_5</zorder>
   <zorder>btn_num_3</zorder>
   <zorder>btn_num_1</zorder>
   <zorder>btn_num_2</zorder>
   <zorder>btn_sum</zorder>
   <zorder>btn_div</zorder>
   <zorder>btn_mul</zorder>
   <zorder>btn_sub</zorder>
   <zorder>btn_sin</zorder>
   <zorder>btn_cot</zorder>
   <zorder>btn_tan</zorder>
   <zorder>btn_cos</zorder>
   <zorder>btn_log</zorder>
   <zorder>btn_sqrt</zorder>
   <zorder>btn_percent</zorder>
   <zorder>btn_ce</zorder>
   <zorder>btn_sqr</zorder>
   <zorder>btn_plus_minues</zorder>
   <zorder>btn_dot</zorder>
   <zorder>btn_num_0</zorder>
   <zorder>btn_result</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
